_projects/2023/100483064/100483064.Rmd

```diff
@@ -0,0 +1,440 @@
+---
+title: "Graph: La evolución del despligue militar de EE.UU."
+description:
+  "This project will explain the process of replicating the graphic La evolución del despliegue militar de EE.UU. created by El Orden Mundial."
+categories: "2023"
+author: Clara Fernanda Martín Villaescusa
+date: "`r Sys.Date()`"
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true 
+---
+
+# REPLICATION
+
+## DATA BASE
+
+```{r}
+library(tidyverse)
+library(scales)
+library(here)
+library(haven)
+library(ggtext)
+library(troopdata)
+library(glue)
+library(ggtext)
+library(ggplotify)
+library(treemapify)
+library(ggalluvial)
+library(ggpmisc)
+library(ggrepel)
+library(patchwork)
+library(scico)
+library(ggbeeswarm)
+library(peacesciencer)
+library(kableExtra)
+library(gridExtra)
+library(ggpmisc)
+library(dplyr)
+#install.packages("Hmisc")
+library(Hmisc)
+
+#install.packages("showtext")
+library(showtext)
```

Comment:
Do you use **all** this libraries? If not, please remove the unused ones.

Also please remove the `install.packages` comments.